.gitignore

```diff
@@ -3,4 +3,5 @@ pbnj
 .idea*/**
```

Comment:
please you squash this commit into "add functional testing"

Reply:
sorry, you're going to have to be more specific. 
![image](https://user-images.githubusercontent.com/12081036/96893720-3daed180-1448-11eb-8130-f6569e8d00ab.png)

Reply:
wow my grammar there :D
Squash 2a9a056 into 89de5bc1

Reply:
updated